roles/maintenance_10_linux/defaults/main.yml

```diff
@@ -29,7 +29,8 @@ linux_additional_allowed_login_ips: []
 
 linux_large_logfiles_prune:
   - /mnt
-
+  - /var/log/lastlog
```

Comment:
Actually, come to think of it, this belongs in `linux_allowed_large_files`, not in `linux_large_logfiles_prune`. Sorry for not noticing this earlier.